.github/workflows/atlas_build-and-push-docker.yml

```
@@ -0,0 +1,203 @@
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+    branches: [main]
+    paths:
+      - 'atlas/AtlasMl/**'
+      - '.github/workflows/atlasml_test-and-build.yml'
+
+  workflow_call:
+    inputs:
+        image-name:
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 8

(indentation)